src/vivarium/framework/values.py

```diff
@@ -170,6 +172,29 @@ def union_post_processor(values: List[NumberLike], _) -> NumberLike:
     return joint_value
 
 
+def step_size_post_processor(values: List[NumberLike], _) -> pd.Series:
```

Comment:
This post-processor is very specific to step-size calculation (especially in its docstring), so it doesn't seem likely to be of general utility. This suggests to me that the post-processor should be defined in the `SimulationClock`, not here in `values.py`.